llama_stack/providers/remote/vector_io/pgvector/pgvector.py

```diff
@@ -70,26 +75,122 @@ def load_models(cur, cls):
     return [TypeAdapter(cls).validate_python(row["data"]) for row in rows]
 
 
+"""
+Three implementations of search for PGVectoIndex:
```

Comment:
Can we add this related documentation in https://github.com/varshaprasad96/llama-stack/blob/main/llama_stack/providers/registry/vector_io.py, so that it gets auto-generated in https://github.com/varshaprasad96/llama-stack/tree/main/docs/source/providers/vector_io